frontend/packages/dev-console/src/utils/pipeline-utils.ts

```diff
@@ -249,3 +250,43 @@ export const containerToLogSourceStatus = (container: ContainerStatus): string =
   }
   return LOG_SOURCE_RUNNING;
 };
+
+type CurrentPipelineStatus = {
+  currentPipeline: Pipeline;
+  status: string;
```

Comment:
is `status` a known enum?

Reply:
Not from its source. I echo'ed the type that was returned by these methods "[pipelineRunFilterReducer](https://github.com/openshift/console/blob/master/frontend/packages/dev-console/src/utils/pipeline-filter-reducer.ts#L3-L32)"